ispyb_lib.py

```diff
@@ -4,26 +4,30 @@
 import time
 import mysql.connector
 import logging
+
 logger = logging.getLogger(__name__)
 
 conn = ispyb.open("/etc/ispyb/ispybConfig.cfg")
 cnx = conn.conn
 cursor = cnx.cursor()
 core = ispyb.factory.create_data_area(ispyb.factory.DataAreaType.CORE, conn)
 
+
 def queryDB(q):
-  cursor.execute(q)
-  try:
-    return list(cursor.fetchall())
-  except TypeError:
-    return 0
+    cursor.execute(q)
+    try:
+        return list(cursor.fetchall())
+    except TypeError:
+        return 0
```

Comment:
Curious under what circumstances is a TypeError raised? From my reading fetchall() should always return an iterable so should be convertable to an empty list